runhouse/resources/hardware/cluster.py

```diff
@@ -184,8 +196,8 @@ def config_for_rns(self):
         )
         if self.is_up():
             creds = (
-                self._resource_string_for_subconfig(self._creds)
-                if self._creds
+                self._resource_string_for_subconfig(self.ssh_creds)
```

Comment:
should be self._ssh_creds, since self.ssh_creds should return only the values of the secret